src/content/docs/browser/new-relic-browser/troubleshooting/troubleshoot-graphql-collection.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: GraphQL metadata is not found on AjaxRequests
+type: troubleshooting
+tags:
+  - Browser
+  - Browser monitoring
+  - Troubleshooting
+metaDescription: Troubleshooting for not seeing GraphQL metadata on AjaxRequest events for your browser app.
+redirects:
+  - /docs/browser/new-relic-browser/troubleshooting/troubleshoot-graphql-collection
+---
+
+## Problem
+
+You are seeing AjaxRequest events, but they do not contain the following GraphQL metadata properties for your browser app.
+* [Operation Framework](https://docs.newrelic.com/attribute-dictionary/?event=AjaxRequest&attribute=operationFramework)
+* [Operation Name](https://docs.newrelic.com/attribute-dictionary/?event=AjaxRequest&attribute=operationName)
+* [Operation Type](https://docs.newrelic.com/attribute-dictionary/?event=AjaxRequest&attribute=operationType)
+
+## Cause
+
+The Browser Agent analyzes each AJAX request body and query and attempts to identify common GraphQL patterns. When detecting these patterns, it will append extra metadata properties to the outgoing AjaxRequest event being harvested. If the agent fails to detect these patterns, it will not add the extra event properties.
```

Comment:
```suggestion
The browser agent analyzes each AJAX request body and query and attempts to identify common GraphQL patterns. When detecting these patterns, it will append extra metadata properties to the outgoing AjaxRequest event being harvested. If the agent fails to detect these patterns, it will not add the extra event properties.
```